.github/workflows/tests-smoke.yaml

```diff
@@ -68,6 +68,18 @@ jobs:
         with:
           persist-credentials: false
 
+      - name: Helm Chart Smoke Test Common label
+        run: |
+          echo "Checking Helm templates for common label"
+
+          grep -RL --exclude validate.yaml .Values.commonLabels install/kubernetes/cilium/templates/*.yaml install/kubernetes/cilium/templates/**/*.yaml >> missing_label
+          
+          if [ -s missing_label ];then
+            echo "These files are missing the .Values.commonLabels, please add them and submit your changes"
+            cat missing_label
+            exit 1
+          fi
+          
```

Comment:
This is no longer needed given that it gets automatically executed by the make invocation below.

Reply:
so just change it to make -C install/kubernetes lint ? I dont see where it make lint for the helm chart is being done in CI?